### ISSUE TITLE
Keyboard Help Dialogue descriptions are not shown in a screenshot from the screenshot wrapper

### ISSUE THREAD

**Issue by phet-steele** [phetsims/john-travoltage#190]
Presumably not a big problem since it seems like an ultra rare use case. Taking a chance this is a sim bug, and not a PhET-iO bug, so reporting it in this repo. However, I am unaware how you would get a screenshot of this dialogue without PhET-iO, so adding that label:

<img width="709" alt="screen shot 2017-02-08 at 12 01 32 pm" src="https://cloud.githubusercontent.com/assets/5863899/22752662/9927c854-edf6-11e6-965c-6b9647d05dd2.png">

Seen on macOS 10.12.3 Chrome. For phetsims/tasks/issues/780.

**Comment by samreid** [comment]
Those are rendered with scenery HTMLText, so I suspect this should become a scenery issue to the tune of "HTMLText not captured in screenshot".  @phet-steele can you please take a quick look in the scenery issues to see if there is already something like that?  If not, can you please create a new issue with this information?

**Comment by phet-steele** [comment]
Moved to phetsims/scenery/issues/605
